spec/Flatten.coffee

```diff
@@ -93,6 +94,11 @@ describe 'Flatten', ->
             src: 'http://timenewsfeed.files.wordpress.com/2012/02/slanglol.jpg?w=480&amp;h=320&amp;crop=1'
             caption: 'Tom Turley / <a href="http://www.gettyimages.com/">Getty Images</a>'
             html: "<figure><img src=\"http://timenewsfeed.files.wordpress.com/2012/02/slanglol.jpg?w=480&amp;h=320&amp;crop=1\"><figcaption>Tom Turley / <a href=\"http://www.gettyimages.com/\">Getty Images</a></figcaption></figure>"
+          ,
+            type: 'image'
+            src: 'http://a.com/b.jpg'
+            caption: '<h1>Title</h1><p>Desc</p>'
```

Comment:
Why is this block.caption rather than block.metadata.caption? 

Should it be html?